src/extlibs/hrvo/hrvo_agent.h

```diff
@@ -144,11 +144,11 @@ class HRVOAgent : public Agent
         // The velocity of the candidate.
         Vector velocity;
 
-        // The index of the first VelocityObstacle in velocityObstacles_ used to compute
+        // The index of the first VelocityObstacle in velocity_obstacles_ used to compute
         // this candidate.
         int velocityObstacle1_;
 
-        // The index of the second VelocityObstacle in velocityObstacles_ used to compute
+        // The index of the second VelocityObstacle in velocity_obstacles_ used to compute
         // this candidate.
         int velocityObstacle2_;
```

Comment:
These two variables should also be snake case

Reply:
I see thanks (I didn't know I could use regex to search through files so I just did it manually).

I found that `[a-zA-Z][ ][a-z]+[A-Z][a-z]+` worked good for finding camelcase variables, I'll pass it on to the other new members working on similar tasks